ci/windows-diagnostics.ps1

```diff
@@ -1,6 +1,10 @@
 Set-StrictMode -Version latest
 $ErrorActionPreference = 'Stop'
 
+$EachineName = Get-CimInstance -ClassName Win32_OperatingSystem | Select-Object CSName | ForEach{ $_.CSName }
```

Comment:
```suggestion
$MachineName = Get-CimInstance -ClassName Win32_OperatingSystem | Select-Object CSName | ForEach{ $_.CSName }
```

Reply:
This is clearly not ready for review yet, just me playing with the only PowerShell environment I have access to, trying to discover the syntax. Thanks for the pointer though :-)

Reply:
Sure, I just thought I'd comment in case you wasted 15 minutes waiting for the output and then getting nothing. 🙂

Glad you caught it!